### PR TITLE
[runtime] Add Skia support for web

### PR DESCRIPTION
**Pull request by byCedric**
# Why

On Android and iOS, Skia seems to work nicely already. But for web, we need to initialize `canvaskit-wasm`. This adds that support.

# How

- Added Snack-specific initialization in the Runtime, [based on the original web init](https://github.com/Shopify/react-native-skia/blob/5adbade804191deef4380ae42796d3b5f050dad7/package/src/web/WithSkia.tsx)
- Note, we have to use `require` instead of `await import` due to #318  

# Test Plan

~~Will share a snack for staging.~~

Here it is! This should work on Android, iOS, and web: https://staging.snack.expo.dev/@bycedrictest/universal-skia-demo.

> Note
> The Android and iOS tabs are linked to Appetize, which isn't updated yet. Use Expo Go `>=2.25.0`.

### CODE REVIEW

**Comment by byCedric** [comment]
cc @wcandillon

File: runtime/src/NativeModules/CanvasKit.web.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// Types are monkey-patched in runtime/types/canvaskit-wasm.d.ts
+import type { CanvasKit as CanvasKitType } from 'canvaskit-wasm';
+import CanvasKitInit from 'canvaskit-wasm/bin/full/canvaskit';
+import { version } from 'canvaskit-wasm/package.json';
```
Comment (wcandillon) [inline runtime/src/NativeModules/CanvasKit.web.tsx:4]: Haha this is such a cool trick 😅

**Comment by brentvatne** [comment]
is this ready to move to ready for review?

**Comment by byCedric** [comment]
@brentvatne It's reviewable, but I'd need to update the other SDK 46 beta upgrade to non-beta when it's out 😄 So I'd prefer to keep this separate until the SDK 46 upgrade is complete.

**Pull request merged**